apps/verify/translations/src/en/pages.json

```diff
@@ -3,7 +3,7 @@
     "header": "How do you want to sign in?"
   },
   "sign-in-brp": {
-    "header": "Sign in",
+    "header": "Enter your details to sign in",
```

Comment:
I think this change is not needed if you look at the latest Figma page

Reply:
I checked the Figma and the details are correct. do we have different figma

Reply:
I was carried away by the above figma diagram